lab_3_keywords_textrank/main.py

```diff
@@ -81,7 +95,7 @@ def preprocess_text(self, text: str) -> tuple[str, ...]:
             tuple[str, ...]
                 clean lowercase tokens with no stop-words
         """
-        pass
+        return self._remove_stop_words(self._clean_and_tokenize(text))
```

Comment:
to pass a result of a function to another without saving to a variable is a bad style